codespell_lib/data/dictionary.txt

```diff
@@ -6531,7 +6533,7 @@ identifer->identifier
 identifers->identifiers
 identifictaion->identification
 identifyable->identifiable
-identiy->identity
+identiy->identify,identity,
```

Comment:
@larsoner shouldn't there be a space between `identify,identity,` ?

Reply:
Not sure if it's required, can you check, and if so open a PR to fix and improve Travis?

Reply:
@luzpaz Good catch! You're right - items with multiple corrections are indeed separated by comma+space in `dictionary.txt`. But this is neither enforced by any test not does it seem to have any effect (other that being more plesant to read in the error report).